geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/SimpleFeatureValidator.scala

```diff
@@ -0,0 +1,105 @@
+package org.locationtech.geomesa.convert
+
+import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
+
+trait SimpleFeatureValidator {
+  def name: String
+  def init(sft: SimpleFeatureType): Unit
+  def validate(sf: SimpleFeature): Boolean
+}
+
+object ValidatorLoader {
+  def createValidator(names: Seq[String], sft: SimpleFeatureType): SimpleFeatureValidator = {
+    val validators = names.map {
+      case "none" => NoneValidator
+      case "has-geo" => new HasGeoValidator
+      case "has-dtg" => new HasDtgValidator
+      case "z-index" => new ZIndexValidator
+      case unk       => throw new IllegalArgumentException(s"Unknown validator $unk")
+    }
+    val composite = new CompositeValidator(validators)
```

Comment:
might make things a little clearer if we only create the composite if there is more than 1